contracts/src/deployers/reth/RETHHyperdriveDeployerCoordinator.sol

```diff
@@ -21,11 +20,9 @@ contract RETHHyperdriveDeployerCoordinator is HyperdriveDeployerCoordinator {
     using SafeERC20 for ERC20;
     using FixedPointMath for uint256;
 
-    /// @notice The Rocket Storage contract.
-    IRocketStorage public immutable rocketStorage;
-
     /// @dev The Rocket Token RETH contract.
-    IRocketTokenRETH internal immutable rocketTokenReth;
+    IRocketTokenRETH internal constant rocketTokenReth =
```

Comment:
I think it's still better for this to be an immutable. If we deploy this to a testnet, hardcoding this will be a problem.